src/backend-test/index.ts

```diff
@@ -0,0 +1,236 @@
+   *
+   * @default true
+   */
+  readonly isAWSDynamoDBlEnabled?: boolean
+}
+
+/**
+ * Backend-test template with TypeScript support.
+ *
+ * @pjid ottofeller-backend-test
+ */
+
+export class OttofellerBackendTestProject extends TypeScriptProject implements IWithTelemetryReportUrl {
+  readonly reportTargetUrl?: string
+  readonly reportTargetAuthHeaderName?: string
+
+  constructor(options: OttofellerBackendTestProjectOptions) {
+    super({
+      ...options,
+      bundlerOptions: {},
+      jest: false,
+      eslint: false,
+      projenrcTs: true,
+      projenrcJs: false,
+      name: 'backend-test',
+      packageManager: options.packageManager ?? NodePackageManager.NPM,
+      srcdir: options.srcdir ?? '.',
+
+      tsconfig: {
+        compilerOptions: {
+          declaration: undefined,
+          experimentalDecorators: undefined,
+          inlineSourceMap: undefined,
+          inlineSources: undefined,
+          noEmitOnError: undefined,
+          noUnusedLocals: undefined,
+          lib: undefined,
+          outDir: undefined,
+          rootDir: undefined,
+          stripInternal: undefined,
+          forceConsistentCasingInFileNames: true,
+          isolatedModules: false,
+          module: 'esnext',
+          moduleResolution: TypeScriptModuleResolution.NODE,
+          noEmit: true,
+          noUncheckedIndexedAccess: true,
+          baseUrl: './',
+          target: 'es6',
+          skipLibCheck: true,
+          paths: {
+            '*': ['./*'],
+          },
+        },
+      },
+
+      // In case Github is enabled remove all default stuff.
+      githubOptions: {mergify: false, pullRequestLint: false},
+      buildWorkflow: false,
+      release: false,
+      depsUpgrade: false,
+      pullRequestTemplate: false,
+      deps: [
+        'dotenv',
+        'axios',
+        'jest',
+        'ts-jest',
+        'jest-extended',
+        'prettier',
+        'eslint-plugin-prettier',
+        'eslint-config-prettier',
+      ],
+    })
+
+    // ANCHOR Source code
+    const assetsDir = path.join(__dirname, '..', '..', 'src/backend-test/assets')
+    sampleCode(this, options, assetsDir)
+
+    /*
+     * Clean off the projen tasks and if needed replace them with regular npm scripts.
+     * This way we ensure smooth ejection experience with all the commands visible in package.json
+     * and no need to keep the projen task runner within an ejected project.
+     */
+    const tasksToRemove = [
+      'build',
+      'bump',
+      'clobber',
+      'compile',
+      'package',
+      'post-compile',
+      'post-upgrade',
+      'pre-compile',
+      'release',
+      'test',
+      // eslint-disable-next-line @cspell/spellchecker -- the word is used once, so no need to add it to the dictionary
+      'unbump',
+      'upgrade',
+      'watch',
+      'projen',
+    ]
+
+    tasksToRemove.forEach(this.removeTask.bind(this))
+
+    this.tryRemoveFile('tsconfig.dev.json')
```

Comment:
This file is used by projen, so whenever you run `npx projen` this config is required.